docs/sdk/serverless-signing/uploading.md

```diff
@@ -0,0 +1,230 @@
+# Uploading Your Lit Action
+
+Lit Actions are powerful JavaScript programs that run on a Lit network. This guide covers the two processes for providing your Lit Action code to the Lit network. For more info on what a Lit Action is, go [here](./overview.md.). For a guide on getting started with Lit Actions, go [here](./quick-start.md).
+
+In order for the Lit nodes to be able to run your Lit Action, they need access to it's code. There are two methods of providing your code to the Lit network for execution:
+
+1. Providing a Code String
+2. Uploading to IPFS
+
+## Choosing a Method
+
+##### Providing a Code String
```

Comment:
maybe we should add a few potential drawbacks to using this method as well? Namely the issues when the code string is too long (is there a max size we can reference?)

Reply:
I'd assume the limitation here is defined by the JavaScript maximum string length (9 quadrillion chars).

Would using a code string perhaps limit the transparency for a user on an application?

Reply:
I can't think of more drawbacks. I was wrong about the code string being too long issue, I was thinking about file size limit for encryption

I added:
```
However, it's important to note some potential drawbacks:

- **Increased Network Usage**: Because the entire Lit Action code must be sent with each request, this can lead to higher network usage and potentially slower overall execution times.
- **Transparency** Depending on where/how you make your source code available to your users, users may find it challenging to review and verify the exact code their [Programmable Key Pairs (PKPs)](../../user-wallets/pkps/overview.md) are interacting with.
```